src/Serilog.Sinks.Console/project.json

```diff
@@ -0,0 +1,60 @@
+{
+  "version": "2.0.0-beta-*",
+  "description": "The console sink for Seriolog - Simple .NET logging with fully-structured events",
+  "authors": [ "Serilog Contributors" ],
+  "tags": [ "serilog", "logging", "semantic", "structured", "console" ],
+  "projectUrl": "http://serilog.net",
+  "licenseUrl": "http://www.apache.org/licenses/LICENSE-2.0",
+  "iconUrl": "http://serilog.net/images/serilog-nuget.png",
+  "frameworks": {
+    "net40": {
+      "compilationOptions": {
+        "keyFile": "../../assets/Serilog.snk"
+      },
+      "frameworkAssemblies": {
+        "System.Configuration": ""
+      },
+      "dependencies": {
+        "Microsoft.Bcl.Async": "1.0.168",
+        "Serilog": "2.0.0-beta-*"
+      }
+    },
+    "net45": {
+      "compilationOptions": {
+        "keyFile": "../../assets/Serilog.snk"
+      },
+      "frameworkAssemblies": {
+        "System.Configuration": ""
+      },
+      "dependencies": {
+        "Serilog": "2.0.0-beta-*"
+      }
+    },
+    "dnx451": {
+      "compilationOptions": {
+        "keyFile": "../../assets/Serilog.snk"
+      },
+      "frameworkAssemblies": {
+        "System.Configuration": ""
+      },
+      "dependencies": {
+        "Serilog": "2.0.0-beta-*"
+      }
+    },
+    "dotnet5.4": {
```

Comment:
Is `dotnet5.4` the minimum we can support for this sink? I guess it depends on the `System.Console` package...